src/svg/ds6/icons.svg

```diff
@@ -610,4 +610,52 @@
+    </symbol>
+    <symbol id="icon-large-case" viewBox="0 0 24 24">
+        <path fill="#111820" d="M7.25 1C6.5596 1 6 1.5596 6 2.25V4H4.25C3.5596 4 3 4.5596 3 5.25v15.5c0 .6904.5596 1.25 1.25 1.25H5c0 .5523.4477 1 1 1s1-.4477 1-1h10c0 .5523.4477 1 1 1s1-.4477 1-1h.75c.6904 0 1.25-.5596 1.25-1.25V5.25C21 4.5596 20.4404 4 19.75 4H18V2.25C18 1.5596 17.4404 1 16.75 1h-9.5zM19 20H5v-2h14v2zM16 4H8V3h8v1zM5 8V6h14v2H5zm0 2h14v6H5v-6z"/>
+    </symbol>
+    <symbol id="icon-customize" viewBox="0 0 24 24">
+        <path fill="#111820" d="M9.1707 6C9.5825 7.1652 10.6938 8 12 8s2.4175-.8348 2.8293-2H22c.5523 0 1-.4477 1-1s-.4477-1-1-1h-7.1707C14.4175 2.8348 13.3062 2 12 2s-2.4175.8348-2.8293 2H2c-.5523 0-1 .4477-1 1s.4477 1 1 1h7.1707zM11 5c0-.5523.4477-1 1-1s1 .4477 1 1-.4477 1-1 1-1-.4477-1-1zM21.8293 11C21.4175 9.8348 20.3062 9 19 9s-2.4175.8348-2.8293 2H2c-.5523 0-1 .4477-1 1s.4477 1 1 1h14.1707c.4118 1.1652 1.5231 2 2.8293 2s2.4175-.8348 2.8293-2H22c.5523 0 1-.4477 1-1s-.4477-1-1-1h-.1707zM19 11c.5523 0 1 .4477 1 1s-.4477 1-1 1-1-.4477-1-1 .4477-1 1-1zM6 16c1.3062 0 2.4175.8348 2.8293 2H22c.5523 0 1 .4477 1 1s-.4477 1-1 1H8.8293C8.4175 21.1652 7.3063 22 6 22c-1.3062 0-2.4175-.8348-2.8293-2H2c-.5523 0-1-.4477-1-1s.4477-1 1-1h1.1707c.4118-1.1652 1.523-2 2.8293-2zm0 2c-.5523 0-1 .4477-1 1s.4477 1 1 1 1-.4477 1-1-.4477-1-1-1z"/>
+    </symbol>
+    <symbol id="icon-help" viewBox="0 0 24 24">
+        <path fill="#111820" d="M13 17a1 1 0 10-2 0 1 1 0 002 0zM9.928 9.371C10.152 8.814 10.911 8 12 8c.738 0 1.312.29 1.648.65.337.36.404.737.305 1.034-.117.352-.513.692-1.182 1.07-.096.055-.223.123-.355.194-.2.108-.413.223-.55.305-.133.08-.307.194-.457.339-.13.126-.409.435-.409.908V14a1 1 0 102 0v-1.093c.087-.05.173-.096.274-.149.128-.068.278-.147.482-.263.656-.371 1.712-1.031 2.094-2.179.376-1.126-.008-2.249-.74-3.032C14.375 6.498 13.261 6 12 6 9.89 6 8.515 7.52 8.072 8.629a1 1 0 001.856.742z" />
+        <path fill="#111820" d="M12 24c6.627 0 12-5.373 12-12S18.627 0 12 0 0 5.373 0 12s5.373 12 12 12zm0-2C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10z" />
+    </symbol>
+    <symbol id="icon-history" viewBox="0 0 24 24">
+        <path fill="#111820" d="M5.852 17.12a7.999 7.999 0 0014.14-5.47A8 8 0 005.453 7.403l1.272-.364a1 1 0 11.55 1.924l-3.5 1a1 1 0 01-1.17-.515l-1.5-3a1 1 0 111.79-.894l.596 1.193a9.997 9.997 0 0110.46-4.554A10 10 0 113.881 17.84c-.322-.449-.155-1.064.324-1.34.42-.242.938-.142 1.284.183a1.165 1.165 0 01.133.148l.002.002c.073.097.15.193.227.286z" />
+        <path fill="#111820" d="M12 7a1 1 0 011 1v4.586l1.707 1.707a1 1 0 01-1.414 1.414l-2-2A1 1 0 0111 13V8a1 1 0 011-1z" />
+    </symbol>
+    <symbol id="icon-lightbulb" viewBox="0 0 24 24">
+        <g clip-path="url(#clip0)" fill="#111820">
```

Comment:
This SVG will need cleaning up.

Reply:
Will get new normalized icons from design. This is still in progress